### ISSUE TITLE
Failed to Compile Error

### ISSUE THREAD

**Issue by Pallaav** [insidesherpa/JPMC-tech-task-2-PY3#102]
I am getting this error
![Screenshot (104)](https://user-images.githubusercontent.com/32379917/80767203-02041080-8b65-11ea-8c13-b318abe45ba2.png)
![Screenshot (105)](https://user-images.githubusercontent.com/32379917/80767266-27911a00-8b65-11ea-8e41-071075aee56a.png)
![Screenshot (107)](https://user-images.githubusercontent.com/32379917/80767302-3bd51700-8b65-11ea-9196-f3a2a7be72fa.png)
![Screenshot (108)](https://user-images.githubusercontent.com/32379917/80767314-3ed00780-8b65-11ea-82b6-a3ce03174f1c.png)

npm version: 6.4.1
node version:11.0.0
nvm version : 1.1.7

**Comment by arunsingh03** [comment]
Same Error Goes with me...

**Comment by varshakothari24** [comment]
I have the same error. Did you resolve it?

**Comment by arunsingh03** [comment]
> I have the same error. Did you resolve it?

yes I have resolved the error just replace the module folder from this content on google drive link -https://drive.google.com/file/d/1amF7C-4mRZCi3Ab5xuH0XCRQubp4aa_t/view
and try to open cmd  prompt in admin mode and this time you don't have to type 'npm install' directly start the  npm

**Comment by varshakothari24** [comment]
yeah i did that and after that i got the perspective error.

![error2](https://user-images.githubusercontent.com/35269809/81411911-a804db80-9160-11ea-80a8-642a60591871.PNG)

**Comment by arunsingh03** [comment]
> yeah i did that and after that i got the perspective error.
> 
> ![error2](https://user-images.githubusercontent.com/35269809/81411911-a804db80-9160-11ea-80a8-642a60591871.PNG)

make it check if there are required file in the @jpmorganchase folder once.
 if not try to delete the whole module folder then replace it with the drive content.

![a](https://user-images.githubusercontent.com/64720296/81412460-835d3380-9161-11ea-8360-5dbc647851c1.png)

**Comment by SiddhlodaAyyar** [comment]
Watch this video and if you still get the error comment down in the youtube section and we'll try to solve it.
https://youtu.be/wRAG9i8O_44

**Comment by jun383914** [comment]
https://drive.google.com/file/d/11T9mffKLG1zuNFk7J8Nen-65jt836G8v/view?usp=sharing

Try to download the linked zip file. It's the missing perspective node. Please unzip it and put it under the node_module folder. That's how I fix mine.
